src/components/utils.js

```diff
@@ -8,3 +8,20 @@ export const isValidAddress = address => {
     return false;
   }
 };
+
+export const validateHash = input => {
+  if (input.length === 0) {
+    return "Hash cannot be empty.";
+  } else if (input.length > 66) {
+    return "Length of Merkle Root Hash is too long.";
+  } else if (input.length < 66) {
+    return "Length of Merkle Root Hash is too short.";
+  } else if (input.slice(0, 2) !== "0x") {
+    console.log(input.slice(0, 2));
+    return "Merkle Root Hash must start with 0x";
+  }
+  return "";
+};
+
+export const validateName = input =>
```

Comment:
Return only true/false, let the caller of this function decide what it wants to do.